Tools/Pipeline/Windows/MainView.cs

```diff
@@ -85,7 +103,7 @@ public void Attach(IController controller)
             };
             _controller.OnBuildFinished += invokeUpdateMenus;
             _controller.OnProjectLoading += invokeUpdateMenus;
-            _controller.OnProjectLoaded += invokeUpdateMenus;
+            _controller.OnProjectLoaded += invokeProjectLoaded;
```

Comment:
Add "UpdateTitle" inside "UpdateMenu", it may seem wasteful, but it's not, we need it there for later (updating project dirty in title aka. *)